libs/sdk-ui-tests-e2e/cypress/tools/table.ts

```diff
@@ -208,4 +208,16 @@ export class Table {
         cy.get(".s-header-cell-label span").invoke("width").should("equal", width);
         return this;
     }
+
+    isCellUnderlined(item: string, isUnderlined = true) {
+        const headerCellLable = cy
+            .get(`.s-header-cell-label.gd-pivot-table-header-label`)
+            .contains(item)
+            .trigger("mouseover");
+        if (isUnderlined) {
```

Comment:
use this`headerCellLable.should(isUnderlined? "have.css": "not.have.css" , "text-decoration", "underline");`